internal/wasm/jit/jit_amd64.go

```diff
@@ -3980,7 +3977,6 @@ func (c *amd64Compiler) compileReturnFunction() error {
 	return nil
 }
 
-// TODO: this is deletable.
```

Comment:
turns out it's not that simple to delete this so removed the comment in this PR 😄